src/storage/src/storage_state.rs

```diff
@@ -711,23 +711,7 @@ impl<'w, A: Allocate> Worker<'w, A> {
                             sink_description,
                         ));
                     }
-
-                    // Continue with other commands.
-                    return;
-                }
-
-                // Suspensions might come in for source exports; they are suspended and
-                // restarted alongside their primary sources.
-                if self
-                    .storage_state
-                    .ingestions
-                    .values()
-                    .any(|v| v.source_exports.contains_key(&id))
-                {
-                    return;
                 }
-
-                panic!("got InternalStorageCommand::SuspendAndRestart for something that is not a source or sink: {id}");
```

Comment:
if you want, we can leave an `error!` or `warn!` logline and get a sentry event to track this issue

Reply:
Good idea, added!